### DISCUSSION TITLE
更换专业版账户登录邮箱后，账户变成一个月的专业账户了，我是才续费的一年，请把时间给我加上去

### DISCUSSION THREAD

**Discussion by WeiChenUm** [getlantern/lantern-forum-cn#845] [General]
一个月前续费的专业账户，但是由于邮箱收验证码限制问题更换了邮箱，用新邮箱登录后发现专业账户变成了一个月的时间，应该是13个月时间才对，应该到2025年11月7号才对，麻烦请把时间给我加回来,我是8月3号续费的一年，加上之前的一个月

![355053925-2900b095-9423-4bf1-9fcf-22c7dc1c61f6](https://github.com/user-attachments/assets/836d55e9-a3f9-49b3-8504-cf2cbda19b9f)

**Comment by MaxMKMax** [comment]
@WeiChenUm 您好. 好的, 我已经帮您提交问题上去了. 支付团队会帮您解决的. 谢谢您支持蓝灯. 感恩! 感谢!
